project/Versions.scala

```diff
@@ -0,0 +1,13 @@
+object Versions {
+  val antlrVersion = "4.9.3"
+  val deltaVersion = "3.2.1"
+  val jacksonVersion = "2.15.0"
```

Comment:
There were two jackson versions used before: 2.15.0 and 2.17.0. This effectively downgrades the version for part of the targets, we'll need to define two separate jackson versions. I'd let unifying versions used for a later change, this may create more issues than it solves